content/layouts/page-breadcrumbs.html

```diff
@@ -17,12 +17,6 @@
     {% when '3-panel-filters' %}
        {% include "content/includes/3-panel-filters.html" %}
 
-    {% when '4-action-page' %}
-      {% include "content/includes/4-action-page.html" %}
-
-    {% when '4-action-page-pending' %}
-      {% include "content/includes/4-action-page-pending.html" %}
-
     {% when '7-get-page' or '7-get-page-alt' %}
```

Comment:
Since the two `7-get-page` includes are being deleted, can they be removed from the breadcrumbs too?

Reply:
Good call!